examples/aggregated/slurm/benchmark/submit.py

```
@@ -0,0 +1,285 @@
+        if not config_files:
+            print(f"No YAML files found in directory: {args.dir}", file=sys.stderr)
+            sys.exit(1)
+        print(f"Found {len(config_files)} YAML file(s) in {args.dir}")
+
+    for config_file in config_files:
+        print(f"Processing: {config_file}")
+        try:
+            config = load_config(config_file)
+            submit_job(config, config_file, args.log_dir, args.dry_run)
+            print(f"Successfully submitted job for: {config_file}\n")
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)